tutorials/p-isa-tools/traces/fhe_complex_single_ops/add_16384_l13_m14_program_trace.csv

```diff
@@ -0,0 +1,2 @@
+instruction,scheme,poly_modulus_degree,rns_terms,cipher_degree,arg0,arg1,arg2,arg3,arg4,arg5,arg6,arg7,arg8,arg9
+add,BGV,16384,14,ct2-2-13,ct0-2-13,ct1-2-13
```

Comment:
The number of attributes before args# does not match number of values before vars ct#-#-#

Reply:
Let's check with @kylanerace. He mentioned some fix for this.